File: src/com/ashitsathish/coding/easy/reverseString/reverseString.js
function reverseString(x) {
    var a = 0;
    var b = x.length - 1;
    var s = x.split("");
    while (a < b) {
        var temp = s[a];
        s[a] = s[b];
        s[b] = temp;
        a++;
        b--;
    }
    console.log(s.join(""));
}


reverseString("ashit");

File: src/com/ashitsathish/coding/easy/leetcode/reverseString/ReverseString.java
package com.ashitsathish.coding.easy.leetcode.reverseString;

public class ReverseString {
    public static void main(String[] args) {
        reverseString("ashit");
    }

    private static void reverseString(String s) {
        int x = 0;
        int y = s.length() -1;
        char[] c= s.toCharArray();
        while (x<y){
            char temp = c[x];
            c[x] = c[y];
            c[y] = temp;
            x++;
            y--;
        }
        String s1 = new String(c);
        System.out.println(s1);
    }
}
